entrypoint.sh

```diff
@@ -68,15 +70,22 @@ changelog=${changelog:-false}
 preset=${preset:-conventionalcommits}
 outputtype=${outputtype:-stdout}
 repopath=${repopath:-./}
+bumpoverride=${bumpoverride:-}
```

Comment:
I think the linting workflow doesn't like this no-default value.

You can lint locally to verify, find instructions in the [contributing guidelines](https://github.com/TomerFi/version-bumper/blob/master/.github/CONTRIBUTING.md#lint-sources)

Reply:
```
2022-08-01 12:06:49 [INFO]   File:[/tmp/lint/entrypoint.sh]
2022-08-01 12:06:49 [INFO]    - File:[entrypoint.sh] was linted with [shfmt] successfully
```
Locally no linting errors are given 😕

Reply:
yes, my bad the linting workflow was screaming about the following redundant empty line and not this statement:

![image](https://user-images.githubusercontent.com/28388442/182147267-57fe2de3-5654-410d-8f1d-a23c4b8c52ce.png)